java-racingcar/src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,51 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import static org.assertj.core.api.Assertions.*;
+
+public class StringTest {
+
+    @Test
+    @DisplayName("문자열을 쉼표로 분리하여 배열로 반환")
+    void splitString() {
+        String input = "1,2";
+        String[] result = input.split(",");
+        assertThat(result).contains("1", "2");
+        assertThat(result).containsExactly("1", "2");
+    }
+
+    @Test
+    @DisplayName("단일 문자열을 쉼표로 분리")
+    void splitSingleString() {
+        String input = "1";
+        String[] result = input.split(",");
+        assertThat(result).contains("1");
+        assertThat(result).containsExactly("1");
```

Comment:
```suggestion
        assertThat("1".split(",")).containsExactly("1");
```
극단적으로 한 라인으로 구현해 보는 것은 어떨까?